ClearKeep/Scenes/FogotPassWord/FogotPasswordView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  FogotPasswordView.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 22/03/2022.
+//
+
+import SwiftUI
+import Common
+import CommonUI
+
+private enum Constants {
+	static let imageScale = 40.0
+}
+
+struct FogotPasswordView: View {
+	// MARK: - Constants
+	private let inspection = ViewInspector<Self>()
+
+	// MARK: - Variables
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.colorScheme) var colorScheme
+	@Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+	@Binding var customServer: CustomServer
+	@State private var showAlert: Bool = false
+	@State private var isLoading = false
+	@State private var isShowError: Bool = false
+	@State private var error: FogotPasswordViewError?
+	@State private(set) var loadable: Loadable<Bool> = .notRequested {
+		didSet {
+			switch loadable {
+			case .loaded:
+				isLoading = false
+				self.showAlert = true
+			case .failed(let error):
+				isLoading = false
+				self.error = FogotPasswordViewError(error)
+				self.isShowError = true
+			case .isLoading:
+				isLoading = true
+			default: break
+				
+			}
+		}
+	}
+
+	// MARK: - Body
+	var body: some View {
+		notRequestedView
+			.onReceive(inspection.notice) { inspection.visit(self, $0) }
+			.applyNavigationBarPlainStyle(title: "ForgotPassword.Title".localized,
+										  titleColor: titleColor,
+										  leftBarItems: {
+				BackButton(customBack)
+			},
+										  rightBarItems: {
+				Spacer()
+			})
+			.progressHUD(isLoading)
+			.hideKeyboardOnTapped()
+			.grandientBackground()
+			.alert(isPresented: $isShowError) {
+				Alert(title: Text(error?.title ?? ""),
+					  message: Text(error?.message ?? ""),
+					  dismissButton: .default(Text(error?.primaryButtonTitle ?? "")))
+			}
+			.alert("ForgotPassword.CheckYourEmail".localized, isPresented: $showAlert) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/FogotPassWord/FogotPasswordView.swift#L67"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>FogotPasswordView.swift:67</code></td>
    </tr>
  </tbody>
</table>